frontend/src/api/WorkSheetAPI.js

```diff
@@ -5,20 +5,23 @@ function updateWorkSheet(id, credentials) {
   return api.put(`/worksheets/${id}`, credentials)
 }
 
-function getWorkSheets(startDate, endDate, status) {
+async function getWorkSheets(startDate, endDate, status) {
   const minTime = startDate ? moment(startDate).format('yyyy.MM.DD') : null
   const maxTime = endDate ? moment(endDate).format('yyyy.MM.DD') : null
-  return status === 'ALL'
-    ? api.get(`/worksheets/`, {
-        params: { minTime, maxTime },
-      })
-    : api.get(`/worksheets/`, {
-        params: { minTime, maxTime, status },
-      })
+  const response =
```

Comment:
kicsit sok a duplikálás... ha a status nem egyenlő azzal hogy ALL akkor nem lehet a BE-nek elküldeni? akkor az egész egyszerűsödne:
```
//valahogy így
await api.get(`/worksheets/`, {
  params: { minTime, maxTime, status: status ==='ALL' ? status : '' },
})
```

ha nem én akkor is inkább így változtatnám meg, szerintem így szebben nézne ki: 
```
const params = { minTime, maxTime };
if(status === 'ALL') {
   params.status = status
}

const response = await api.get(`/worksheets/`, {params})
return response?.data
```

Reply:
szintén időszűke miatt kérem engedjük el, ígérem megcsináltatom a csapattal egy refaktos issue-ben :laughing:

Reply:
részemről ok, resolváljátok és mehet a merge :)